### PR TITLE
Stop explicitly setting spatial=True to batch norm

### PR DESCRIPTION
**Pull request by shinh**
ONNX opset 9 removed `spatial` attribute of BatchNormalization.

https://github.com/onnx/onnx/pull/1637

Since the default value of `spatial` has been True for the
whole time, just dropping the attribute would be the simplest
and the most robust fix to handle the update.

### CODE REVIEW

**Comment by mitmul** [comment]
LGTM
